x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_view.tsx

```diff
@@ -454,131 +448,154 @@ export const IndexDataVisualizerView: FC<IndexDataVisualizerViewProps> = (dataVi
     () => currentDataView.timeFieldName !== undefined && currentDataView.timeFieldName !== '',
     [currentDataView.timeFieldName]
   );
-  const helpLink = docLinks.links.ml.guide;
-
   return (
-    <Fragment>
-      <EuiPageBody data-test-subj="dataVisualizerIndexPage" paddingSize="none" panelled={false}>
-        <EuiFlexGroup gutterSize="m">
-          <EuiFlexItem>
-            <EuiPageContentHeader className="dataVisualizerPageHeader">
-              <EuiPageContentHeaderSection>
-                <div className="dataViewTitleHeader">
-                  <EuiTitle size={'s'}>
-                    <h2>{currentDataView.getName()}</h2>
-                  </EuiTitle>
-                  <DataVisualizerDataViewManagement
-                    currentDataView={currentDataView}
-                    useNewFieldsApi={true}
-                  />
-                </div>
-              </EuiPageContentHeaderSection>
-
-              <EuiFlexGroup
-                alignItems="center"
-                justifyContent="flexEnd"
-                gutterSize="s"
-                data-test-subj="dataVisualizerTimeRangeSelectorSection"
+    <EuiPageBody data-test-subj="dataVisualizerIndexPage" paddingSize="none" panelled={false}>
+      <EuiFlexGroup gutterSize="m">
+        <EuiFlexItem>
+          <EuiPageContentHeader
+            data-test-subj="dataVisualizerPageHeader"
+            css={
+              compact &&
+              css`
+                flex-direction: column;
+                align-items: flex-start;
+              `
+            }
```

Comment:
can be just an object instead

Reply:
Updated here [`2d35742` (#147137)](https://github.com/elastic/kibana/pull/147137/commits/2d35742e343066904b17034fa60a0b1c3fb5536e)